Project/Engine/src/sysFrame/System/Math/MathUtils.cpp

```
@@ -136,3 +136,16 @@
     std::uniform_real_distribution<float> dist(min, max);
     return dist(random);
 }
+
+float MathUtils::Distance(const Vector3& a, const Vector3& b) {
+    return std::sqrtf(std::powf(a.x - b.x, 2) + std::powf(a.y - b.y, 2) + std::powf(a.z - b.z, 2));
+}
+
+Vector3 MathUtils::TransformNormal(const Vector3& v, const Transform& t) {
```

cppcheck (2.10-2)

[style] The function 'TransformNormal' is never used.

(unusedFunction)